src/Interactions.jl

```diff
@@ -198,30 +214,34 @@ end
 Adds an external field ``𝐁`` with Zeeman coupling,
 
 ```math
-    -∑_i 𝐁 ⋅ 𝐦_i.
+    -∑_i 𝐁 ⋅ 𝐌_i.
 ```
 
-The magnetic moments are ``𝐦_i = μ_B g 𝐬_i`` where ``g`` is the g-factor or
-g-tensor, and the spin magnitude ``|𝐬_i|`` is typically a multiple of 1/2. The
+The magnetic moments are ``𝐌_i = μ_B g 𝐒_i`` where ``g`` is the g-factor or
+g-tensor, and the spin magnitude ``|𝐒_i|`` is typically a multiple of 1/2. The
 Bohr magneton ``μ_B`` is a physical constant, with numerical value determined by
 the unit system.
```

Comment:
In the code above (line 421 in Ewald.jl), it looked like you were using `m ~ S g` and not `m ~ g S`. Should we update this comment?

Do we handle the Bohr magneton yet in the code?

Reply:
Same as above -- that line was constructing the `A` tensor, and just trying to absorb as many factors as possible into it. Bohr magneton to come soon.